doc/wiki/Troubleshooting.md

```diff
@@ -11,4 +11,30 @@
    * **Solution**: Update `CUDA` to 11.3 or higher.
 
 * * *
+* #### Check static array by [AddressSanitizer](https://github.com/google/sanitizers/wiki/AddressSanitizer) (ASen)
+   * **Description**: To detect the wrong usage of the static array.
+   * **Steps**:
+      1. Use `g++` compiler.
+      1. Compile and link with flags `-fsanitize=undefined -fsanitize=address`.
 
+   * **Example of using `eureka` machine**
+      1. Uncomment the following lines in `configs/eureka_gnu.config`
+         ```
+         #CXXFLAG -fsanitize=undefined -fsanitize=address
+         #LIBFLAG -fsanitize=undefined -fsanitize=address
+         ```
+      1. Edit `src/Makefile_base`
+    
+         Add the `$(CXXFLAG)` at the makefile linking part, usually at the bottom of the makefile. The line should be like this:
```

Comment:
Is here necessary to add `$(CXXFLAG)` here? I think the `CXXFLAG` has been set somewhere. I can also successfully run without this.

Reply:
Yes, the `-fsanitize=address` flag is necessary for linking. May I know how do you compile GAMER in your successful run?